articles/user-guide/quickref/iqsharp.md

```diff
@@ -0,0 +1,25 @@
+---
+title: IQ# Magic Commands
+description: Quick reference page for IQ# magic commands with Q# Jupyter Notebooks
+author: gillenhaalb
+ms.author:  a-gibec@microsoft.com
+ms.date: 03/05/2020
+uid: microsoft.quantum.guide.quickref.iqsharp
+---
+
+# IQ# Magic Commands
```

Comment:
As mentioned elsewhere, it'd be good to drop the autogenerated IQ# reference in somewhere, now that that exists.